parachain/runtime/src/lib.rs

```diff
@@ -235,6 +238,64 @@ impl pallet_utility::Config for Runtime {
     type WeightInfo = ();
 }
 
+pub struct RelayChainBlockNumberProvider<T>(sp_std::marker::PhantomData<T>);
+
+impl<T: cumulus_pallet_parachain_system::Config> BlockNumberProvider for RelayChainBlockNumberProvider<T> {
+    type BlockNumber = BlockNumber;
+
+    fn current_block_number() -> Self::BlockNumber {
+        cumulus_pallet_parachain_system::Pallet::<T>::validation_data()
+            .map(|d| d.relay_parent_number)
+            .unwrap_or_default()
+    }
+}
+
+parameter_types! {
+    pub MinVestedTransfer: Balance = 0;
+    pub const MaxVestingSchedules: u32 = 100;
```

Comment:
Seems a bit much?

Reply:
Thanks for flagging this, I actually think this may be too low if there is one vesting schedule per account. I'll verify and update if so.

Reply:
Hmm I thought a schedule would be reused across multiple accounts, but that doesn't seem to be the case, so then it's low, yea. Looking at the code, it iterates over all schedules in a couple of places, which must be why they set a fixed limit. We went really out of our way to prevent iteration in reward/staking.. Do you think iteration is ok here in this case? I am not familiar enough with this to know when and how often these functions are called

Reply:
I can confirm that this is limit is per account, so we should probably use a lower value. For our use case we only need one vesting schedule for the initial airdrop, so iteration is not a concern.

Reply:
Updated the maximum to one for now.